traffic_ops/app/lib/Extensions/TrafficStats/API/CdnStats.pm

```diff
@@ -52,10 +52,10 @@ sub get_usage_overview {
 
 	my ( $rc, $result ) = $cstats->get_usage_overview();
 	if ( $rc == SUCCESS ) {
-		return $self->success($result);
+		return $self->deprecation_with_no_alternative(200, $result);
 	}
 	else {
-		return $self->alert($result);
+		return $self->deprecation_with_no_alternative(500, $result);
```

Comment:
nit but the alert perl method returned a 400 response code instead of a 500 this should match that, once that is changed I believe this PR is ready to go

Reply:
weird that it returned a 400 but good catch. changing.